streamalert_cli/athena/helpers.py

```diff
@@ -43,9 +45,14 @@ def add_partition_statement(partitions, bucket, table_name):
         table_name (str): The name of the Athena table
 
     Returns:
-        str: The ALTER TABLE statement to add the new partitions
+        list[str]: The ALTER TABLE statements to add the new partitions
     """
-    statements = ['ALTER TABLE {} ADD IF NOT EXISTS'.format(table_name)]
+
+    # monitor statement length, if it is less than 0, start a new statement.
+    # return a list of str
+    results = []
```

Comment:
store the initial value outside the list so it can be referenced without recreating:

```
initial_value = 'ALTER TABLE {} ADD IF NOT EXISTS'.format(table_name)
statement = [initial_value]
```

then reference `initial_value` below on line 72